config/go.d/mysql.conf

```diff
@@ -0,0 +1,87 @@
+# netdata go.d.plugin configuration for mysql
+#
+# This file is in YaML format. Generally the format is:
+#
+# name: value
+#
+# There are 2 sections:
+#  - GLOBAL
+#  - JOBS
+#
+#
+# [ GLOBAL ]
+# These variables set the defaults for all JOBs, however each JOB may define its own, overriding the defaults.
+#
+# The GLOBAL section format:
+# param1: value1
+# param2: value2
+#
+# Currently supported global parameters:
+#  - update_every
+#    Data collection frequency in seconds. Default: 1.
+#
+#  - autodetection_retry
+#    Re-check interval in seconds. Attempts to start the job are made once every interval.
+#    Zero means not to schedule re-check. Default: 0.
+#
+#
+# [ JOBS ]
+# JOBS allow you to collect values from multiple sources.
+# Each source will have its own set of charts.
+#
+# IMPORTANT:
+#  - Parameter 'name' is mandatory.
+#  - Jobs with the same name are mutually exclusive. Only one of them will be allowed running at any time.
+#
+# This allows autodetection to try several alternatives and pick the one that works.
+# Any number of jobs is supported.
+#
+# The JOBS section format:
+#
+# jobs:
+#   - name: job1
+#     param1: value1
+#     param2: value2
+#
+#   - name: job2
+#     param1: value1
+#     param2: value2
+#
+#   - name: job2
+#     param1: value1
+#
+#
+# [ List of JOB specific parameters ]:
+#  - dsn
+#    Data Source Name.
+#    Format: [username[:password]@][protocol[(address)]]/dbname[?param1=value1&...&paramN=valueN]
+#    Except for the databasename, all values are optional. So the minimal DSN is: /dbname
+#    If you do not want to preselect a database, leave dbname empty: /
+#    Full description: https://github.com/go-sql-driver/mysql#dsn-data-source-name
+#    Syntax:
+#     dsn: netdata@tcp(127.0.0.1:3306)/
+#
+#
+# [ JOB defaults ]:
+#  No parameters
+#
+#
+# [ JOB mandatory parameters ]:
+#  - name
+#  - dsn
+# ------------------------------------------------MODULE-CONFIGURATION--------------------------------------------------
+# [ GLOBAL ]
+update_every: 1
+autodetection_retry: 0
+#
+#
+# [ JOBS ]
+jobs:
+  - name: local
+    dsn: netdata@tcp(127.0.0.1:3306)/
+  - name: local2
+    dsn: root@/
+  - name: socket
+    dsn: root@unix(/var/run/mysqld/mysqld.sock)/
+  - name: socket2
+    dsn: root@unix(/usr/local/var/mysql/mysql.sock)/
```

Comment:
> #  - Jobs with the same name are mutually exclusive. Only one of them will be allowed running at any time.

We need only one of them be running at a time. Please change name to `local` to all of them.

Reply:
This is the feature - you add several jobs with same name but different parameters and the first that succed will run, all other will be dropped.

https://github.com/netdata/go.d.plugin/blob/140abe8d3ade403813619abc93a193f88e797ad3/godplugin/jobstart.go#L12-L16